src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -31,6 +33,8 @@
  */
 final class EntityValueResolver implements ValueResolverInterface
 {
+    use ValueBagResolverTrait;
```

Comment:
This change will render the Doctrine Bridge incompatible with older versions of the HttpKernel component. That's fine, but we need to tell Composer about that.

Please adjust this conflict rule:

https://github.com/symfony/symfony/blob/9b0ca992923f71f5b6e3571d0fdac736902d3d79/src/Symfony/Bridge/Doctrine/composer.json#L61

Since you're currently targeting 7.2 for this change, declaring incompatibility with versions before 7.2 would solve the problem.